addons/kommander/1.4/kommander.yaml

```
@@ -9,19 +9,19 @@ metadata:
     # This was originally added to support the PVC's needed for the Kubecost subcomponent.
     kubeaddons.mesosphere.io/hack-requires-defaultstorageclass: "true"
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "1.4.1-4"
-    appversion.kubeaddons.mesosphere.io/kommander: "1.4.1"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.4.2-1"
+    appversion.kubeaddons.mesosphere.io/kommander: "1.4.2"
     endpoint.kubeaddons.mesosphere.io/kommander: /ops/portal/kommander/ui
     appversion.kubeaddons.mesosphere.io/thanos: 0.3.21
     appversion.kubeaddons.mesosphere.io/karma: 1.4.1
-    appversion.kubeaddons.mesosphere.io/kommander-grafana: 6.6.0
+    appversion.kubeaddons.mesosphere.io/kommander-grafana: 1.4.2
```

```suggestion
    appversion.kubeaddons.mesosphere.io/kommander-grafana: 6.6.0
```

weird. this shouldn't have been edited